src/components/ui/QuantitySelector/quantity-selector.scss

```diff
@@ -0,0 +1,53 @@
+[data-store-quantity-selector] {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  width: 8.125rem;
+  height: var(--space-7);
+  background-color: var(--bg-neutral-lightest);
+  border: var(--border-width-0) solid var(--color-border-input);
+  border-radius: var(--border-radius-default);
+
+  [data-store-icon] {
+    margin: 0;
+    color: var(--color-border-input-selected);
+    line-height: 0;
+  }
+
+  [data-quantity-selector-input] {
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    text-align: center;
+    background-color: transparent;
+    border-width: 0;
+
+    &:focus { outline: 0; }
+  }
+
+  [data-quantity-selector-button] {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    background-color: transparent;
+    border-color: transparent;
+
+    &:disabled {
+      cursor: not-allowed;
+      [data-store-icon] { color: var(--color-border-input-disabled); }
+    }
+  }
+
+  &[data-store-quantity-selector="disabled"] {
+    background-color: var(--bg-disabled);
+    border-color: var(--bg-disabled);
+  }
```

Comment:
You can remove it from the [data-store-quantity-selector] scope
![image](https://user-images.githubusercontent.com/15680320/147604166-271fb4e3-cc79-410e-baf9-96c93088cf44.png)

```css
[data-store-quantity-selector="disabled"] {
    background-color: var(--bg-disabled);
    border-color: var(--bg-disabled);
  }
```

Reply:
Done at https://github.com/vtex-sites/base.store/pull/183/commits/8e8c09dd280f086eb151151c7c11c94d06410145